starter-repos/app-template/skaffold.yaml

```diff
@@ -11,3 +11,10 @@ deploy:
   kustomize:
     paths:
     - config/envs/dev
+profiles:
+  # Profile used when building images in CI
+  - name: ci
+    build:
+      cluster:
+        dockerConfig:
+          path: ~/.docker/config.json
```

Comment:
Actually, could we move this to skaffold-ci.yaml given that this is only for image building?

Reply:
we can't. The shared-ci-cd specifically requires to use the config under name `skaffold.yaml`. Moving it to skaffold-ci.yaml will cause not found error.

Reply:
We can modify shared-ci-cd.

Reply:
If you're worried that we might affect golang-template, you could create another yaml file under shared-ci-cd and just use that one.